apps/asap-cli/src/scripts/algolia/set-settings.ts

```diff
@@ -33,15 +35,18 @@ export const setAlgoliaSettings = async ({
   await index
     .setSettings({
       ...indexSchema,
-      replicas: [replicaIndexName],
+      ...(hasReverseTimestampReplica ? { replicas: [replicaIndexName] } : {}),
```

Comment:
I think we won't need the reverse timestamp replica for analytics because we don't want to order teams based on created at or something like that